<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmittanceWidget</class>
 <widget class="QWidget" name="EmittanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emittance measurement</string>
  </property>
  <layout class="QHBoxLayout">
   <item>
    <widget class="QGroupBox">
     <property name="title">
      <string>Monitors</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="TableView" name="monitorTable"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox">
     <property name="title">
      <string>Fit results</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="TableView" name="resultsTable"/>
      </item>
      <item>
       <widget class="QWidget" name="optionsWidget" native="true">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QCheckBox" name="longCheckBox">
           <property name="text">
            <string>Match from start</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="dispersionCheckBox">
           <property name="text">
            <string>Dispersion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="couplingCheckBox">
           <property name="text">
            <string>4D</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="updateButton">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyButton">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Ok|QDialogButtonBox::Save</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>madgui/widget/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
